plc4j/drivers/ab-eth/src/main/java/org/apache/plc4x/java/abeth/configuration/AbEthConfiguration.java

```diff
@@ -18,12 +18,10 @@
  */
 package org.apache.plc4x.java.abeth.configuration;
 
-import org.apache.plc4x.java.abeth.AbEthDriver;
 import org.apache.plc4x.java.spi.configuration.Configuration;
 import org.apache.plc4x.java.spi.configuration.annotations.ConfigurationParameter;
-import org.apache.plc4x.java.transport.tcp.TcpTransportConfiguration;
 
-public class AbEthConfiguration implements Configuration, TcpTransportConfiguration {
```

Comment:
instead of separating it you could have just added extends `DefaultTcpTransportConfiguration` but I don't see a issue in having a separate class `AbEthTcpTransportConfiguration`

Reply:
Guess it is for consistency... see comment below on the other class